twitter.html

```diff
@@ -0,0 +1,544 @@
+                </div>
+                <div class="sm_article_img"></div>
+              </div>
+            </li>
+
+            <div
+              class="border show_more trends_item transparent hover_square round_bottom"
+            >
+              Show more
+            </div>
+          </ul>
+        </div>
+
+        <div class="border">
+          <ul class="list round">
+            <div
+              class="border right_panel_heads trends_item right_items_padding transparent"
+            >
+              <h3 class="transparent">Who to follow</h3>
+            </div>
+
+            <li class="list border hover_square">
+              <div class="border account right_items_padding transparent">
+                <div class="border account-icon"></div>
+                <div class="border account-details transparent">
+                  <h4 class="border transparent">.tyzden</h4>
+                  <p class="border at_account transparent">@tyzdensk</p>
+                </div>
+                <p class="border follow_button">Follow</p>
+              </div>
+            </li>
+
+            <li class="list border hover_square">
+              <div class="border account right_items_padding transparent">
+                <div class="border account-icon"></div>
+                <div class="border account-details transparent">
+                  <h4 class="border transparent">.tyzden</h4>
+                  <p class="border at_account transparent">@tyzdensk</p>
+                </div>
+                <p class="border follow_button">Follow</p>
+              </div>
+            </li>
+
+            <li class="list border hover_square">
+              <div class="border account right_items_padding transparent">
+                <div class="border account-icon"></div>
+                <div class="border account-details transparent">
+                  <h4 class="border transparent">.tyzden</h4>
+                  <p class="border at_account transparent">@tyzdensk</p>
+                </div>
+                <p class="border follow_button">Follow</p>
+              </div>
+            </li>
+
+            <li
+              class="list border show_more hover_square round_bottom transparent"
+            >
+              Show more
+            </li>
+          </ul>
+        </div>
+
+        <div class="border">
+          <ul class="border terms_list">
+            <li class="border sm_grey hover_underline">Terms of Service</li>
+            <li class="border sm_grey hover_underline">Privacy Policy</li>
+            <li class="border sm_grey hover_underline">Cookie Policy</li>
+            <li class="border sm_grey hover_underline">Accessibility</li>
+            <li class="border sm_grey hover_underline">Ads info</li>
+            <li class="border sm_grey hover_underline">More</li>
+            <br />
+            <li class="border hover_underline sm_grey">© 2022 Twitter, Inc.</li>
+          </ul>
```

Comment:
You could maybe use the `sm_grey` class on the `ul` directly instead of repeating it on every li ? Or is there a reason for it ?

Reply:
and you are right again...i just changed it and it works that way too.